main.py

```diff
@@ -6,23 +6,32 @@
 # Global variable used for storing data (bad practice)
 data = []
```

Comment:
**Suggestion:** Use a database or a more efficient data structure instead of a list to store items.
```suggestion
# Use a database or a more efficient data structure to store items
data = {}

```